src/backend/src/routes/projects.routes.ts

```diff
@@ -86,7 +86,7 @@ projectRouter.post(
   nonEmptyString(body('manufacturerName')),
   nonEmptyString(body('manufacturerPartNumber')),
   nonEmptyString(body('pdmFileName').optional()),
-  intMinZero(body('quantity')),
+  body('quantity'),
```

Comment:
we should make a new validation method for decimals, see how we do it in validation.utils

Reply:
What should it validate? like the fact that its a decimal greater than 0?

Reply:
yup